abilian/sbe/apps/communities/views/wizard.py

```diff
@@ -91,27 +92,44 @@ def wizard_extract_data(emails, is_csv=False):
     return existing_accounts_objects, existing_members_objects, accounts_list
 
 
-def wizard_read_csv(csv):
+def wizard_read_csv(csv=None):
     file_extension = secure_filename(csv.filename).split(".")[-1]
-    if file_extension == "csv":
-        contents = csv.readlines()
-        new_accounts = []
-        for line in contents:
-            account = {}
-            data = line.split(";")
-            if len(data) == 4:
-                account["email"] = data[0].strip()
-                account["first_name"] = data[1].strip()
-                account["last_name"] = data[2].strip()
-                account["role"] = data[3].strip()
-                new_accounts.append(account)
-        return new_accounts
-    return False
+
+    if file_extension != "csv":
+        return False
```

Comment:
(minor comment) maybe it's better to return a `[ ]` ? So than the type is consistent: better for reading, the calling method could check the length,…